azure-pipelines-integration-scouting.yml

```diff
@@ -0,0 +1,14 @@
+# Run branch daily at midnight UTC
```

Comment:
is it possible to setup some notifications when this fails?

Reply:
Not sure another email from AzDO will make things more visible. I did add a build status indicator to the tiger dashboard, which should be checked daily-ish.

Reply:
yeah - I would want a notification (maybe to infraswat channel or something) if we could trigger this only when the queue changes.  But if it's triggering every night maybe not